### PR TITLE
Randomize arrival flight levels

### PR DESCRIPTION
**Pull request by pikachu1992**
- Works to LPMA

euroscope_b5ifr4qjic

### CODE REVIEW

**Comment by pikachu1992** [comment]
- works with LPFR

euroscope_asjaalqoxl

**Comment by pikachu1992** [comment]
- Works on LPPT

euroscope_spyd1smxpl

**Comment by pikachu1992** [comment]
- https://github.com/Portugal-vAcc/es-scenario/pull/25/commits/adacea2def6fbc72f35a56bd05d3f15dba5cad50

euroscope_spsnb7omad

**Comment by pikachu1992** [comment]
- #26

File: src/flight.py [hunk 1]
```diff
@@ -16,20 +16,31 @@ def departure(callsign, departure, destination, stand):
   )
 
 def arrival(callsign, departure, destination, rwy):
-  flight_plan, position = choice(AIRPORT_SETTINGS[destination]['ARRIVAL_FPL'][departure])
+  flight_plan_route, position, level, fp_direction = choice(AIRPORT_SETTINGS[destination]['ARRIVAL_FPL'][departure])
   route, position_coords, expected_alt = AIRPORT_SETTINGS[destination]['ARRIVAL_ROUTES'][rwy][position]
-
+  random_altitude = choice(AIRPORT_SETTINGS[destination]['GET_FL'][fp_direction][level])
+  flight_plan = ':*A:I:B738:364:'+departure+':0000:0000:'+random_altitude+':'+destination+':00:00:0:0::/v/:'
```
Comment (pedro2555) [inline src/flight.py]: same story, use str.format()

File: src/flight.py [hunk 2]
```diff
@@ -16,20 +16,31 @@ def departure(callsign, departure, destination, stand):
...
+@N:{callsign}:0000:1:{position_coords}:{random_altitude}:0:50:0:0
 $FP{callsign}{flight_plan}
 $ROUTE:{route}
 DELAY:3:7
 REQALT:{position}:{expected_alt}
 '''.format(
     callsign=callsign,
     position_coords=position_coords,
-    flight_plan=flight_plan,
+    flight_plan=flight_plan+flight_plan_route,
```
Comment (pedro2555) [inline src/flight.py]: str.format() use params

**Pull request merged**